testing/trino-product-tests/src/main/java/io/trino/tests/product/deltalake/TestDeltaLakeColumnMappingMode.java

```diff
@@ -65,26 +65,16 @@ public void testColumnMappingModeNone()
     @Test(groups = {DELTA_LAKE_DATABRICKS, DELTA_LAKE_EXCLUDE_73, DELTA_LAKE_EXCLUDE_91, PROFILE_SPECIFIC_TESTS})
     public void testColumnMappingModeId()
     {
-        String tableName = "test_dl_column_mapping_mode_id" + randomTableSuffix();
-
-        onDelta().executeQuery("" +
-                "CREATE TABLE default." + tableName +
-                " (a_number INT)" +
-                " USING delta " +
-                " LOCATION 's3://" + bucketName + "/databricks-compatibility-test-" + tableName + "'" +
-                " TBLPROPERTIES ('delta.columnMapping.mode'='id')");
-
-        try {
-            assertQueryFailure(() -> onTrino().executeQuery("SELECT * FROM delta.default." + tableName))
-                    .hasMessageContaining("Only 'name' or 'none' is supported for the 'delta.columnMapping.mode' table property");
-        }
-        finally {
-            onDelta().executeQuery("DROP TABLE default." + tableName);
-        }
+        testColumnMappingMode("id");
     }
 
     @Test(groups = {DELTA_LAKE_DATABRICKS, DELTA_LAKE_OSS, DELTA_LAKE_EXCLUDE_73, DELTA_LAKE_EXCLUDE_91, PROFILE_SPECIFIC_TESTS})
     public void testColumnMappingModeName()
+    {
```

Comment:
Can we please have a rename test for a partitioning column ?

I want to make sure that the reading from the delta table still works after doing the rename with the `delta.column.mapping` set to `id` / `name`.

Reply:
Added a partition column and rename statement.